Src/Microsoft.Dynamic/zlib.net/Inflate.cs

```diff
@@ -487,7 +487,7 @@ internal int inflateSetDictionary(ZStream z, byte[] dictionary, int dictLength)
 		{
 			int index = 0;
 			int length = dictLength;
-			if (z == null || z.istate == null || z.istate.mode !=  InflateMode.DICT0)
+			if (z?.istate?.mode != InflateMode.DICT0)
```

Comment:
All these changes make me want to go back in my SW at work and actually use new features of C# :-)